core/src/main/java/com/exadel/etoolbox/backpack/core/services/pckg/UploadPackageService.java

```diff
@@ -24,10 +23,10 @@ public interface UploadPackageService {
     /**
      * Method upload package by byte[] array representation
      *
-     * @param session              {@code Session} instance used to adapt to JcrPackageManager object.
-     * @param fileUploadBytesArray {@code byte[]} instance used provides the content of the package.
+     * @param resourceResolver     {@code ResourceResolver} instance used to adapt to Session object and access resources
+     * @param fileUploadBytesArray {@code byte[]} instance used provides the content of the package
```

Comment:
(cosmetic) Just `Content of the package`. Can also add a not on possible restrictions, like: can it be null? can it be empty?